test/test_heatmap.py

```diff
@@ -29,6 +29,45 @@ def test_heatmap_default():
     assert "Monday" in html_content
 
 
+def test_heatmap_display_yaxis_label():
+    data = [[i, j, random.randint(0, 50)] for i in range(24) for j in range(7)]
+    heatmap = HeatMap()
+    heatmap.add(
+        "热力图直角坐标系",
+        X_TIME,
+        Y_WEEK,
+        data,
+        is_visualmap=True,
+        is_label_show=False,
+        visual_text_color="#000",
+        visual_orient='horizontal',
+    )
+    html_content = heatmap._repr_html_()
+    assert r'"formatter": "{value} "' not in html_content
+    assert "Saturday" in html_content
+    assert "Monday" in html_content
+
+
+def test_heatmap_yaxis_formatter():
+    data = [[i, j, random.randint(0, 50)] for i in range(24) for j in range(7)]
+    heatmap = HeatMap()
+    heatmap.add(
+        "热力图直角坐标系",
+        X_TIME,
+        Y_WEEK,
+        data,
+        is_visualmap=True,
+        is_label_show=False,
+        visual_text_color="#000",
+        visual_orient='horizontal',
+        yaxis_formatter="",
+    )
+    html_content = heatmap._repr_html_()
+    assert r'"formatter": "{value} "' in html_content
```

Comment:
looks like  with or without yaxis_formatter, the assertion is the same.

Reply:
`"formatter": "{value} "` will not included to `"axisLabel"` in the HTML if `yaxis_formatter` not specified at the `heatmap.add` method call, in this case, y-axis labels are properly displayed.
On the other hand, `"formatter": "{value} "` will included in the HTML if `yaxis_formatter` specified other than `None`, even if the value is empty string(`""`) . This case, y-axis displays y-values (0-6).
Does it makes sense?

Reply:
Alright, so to get week day names display, yaxis_formatter should be given "".